server/container_stop.go

```diff
@@ -38,6 +38,12 @@ func (s *Server) StopContainer(ctx context.Context, req *types.StopContainerRequ
 		return err
 	}
 
+	if s.config.EventedPLEG {
+		err := s.generateCRIEvent(ctx, c, types.ContainerEventType_CONTAINER_DELETED_EVENT)
+		if err != nil {
```

Comment:
nit: these can be compressed:
`if err := ...; err != nil`

Reply:
Addressed